index.html

```diff
@@ -29,13 +36,13 @@
   <main>
     <section class="headline">
       <div class="header-container">
-        <h1>Hey there. I’m Tofunmi</h1>
-        <h2> I’m a software developer</h2>
+        <h1>Hey there. I'm Tofunmi</h1>
+        <h2> I'm a software developer</h2>
       </div>
       <p class="primary-content"> I can help you build a product , feature or website Look through some of my work
         and
         experience! If you like what you see and have a project you need coded,
-        don’t hestiate to contact me.</p>
+        don't hestiate to contact me.</p>
 
       <ul class="social-media-icons">
```

Comment:
Please use all background images from the design and place them in the right place. Kindly see the screenshot below for reference.
![intro](https://user-images.githubusercontent.com/35504149/131216467-dad63197-bd86-4da2-81c0-d1c4eeb27e46.png)